clang/test/Driver/sycl-ftarget-register-alloc-mode-old-model.cpp

```diff
@@ -20,6 +20,14 @@
 // RUN:    -fsycl-targets=spir64_gen -Xs "-device pvc" %s 2>&1 \
 // RUN:   | FileCheck %if system-windows %{ -check-prefix=DEFAULT_AOT %} %else %{ -check-prefix=AUTO_AOT %} %s -DDEVICE=pvc
 
+// RUN: %clang -### -fsycl --no-offload-new-driver \
+// RUN:    -fsycl-targets=spir64_gen -Xsycl-target-backend "-device pvc" %s 2>&1 \
+// RUN:   | FileCheck %if system-windows %{ -check-prefix=DEFAULT_AOT %} %else %{ -check-prefix=AUTO_AOT %} %s -DDEVICE=pvc
+
+// RUN: %clang -### -fsycl --no-offload-new-driver \
```

Comment:
Any reason for these 2 RUN commands to be exactly the same?

Reply:
These aren't the same - one is using `-Xsycl-target-backend "-device pvc"` and the other is using `-Xsycl-target-backend=spir64_gen "-device pvc"`.  Just covering all the bases here.

Reply:
Thanks, I overlooked the one without the equals part. 🙃